atest/robot/libdoc/LibDocLib.py

```diff
@@ -80,3 +80,13 @@ def get_repr_from_json_arg_model(self, model):
                                name=model['name'],
                                types=tuple(model['types']),
                                default=model['default'] or ArgInfo.NOTSET))
+
+    def is_typing_extensions(self):
+        try:
+            from typing_extensions import TypedDict
+            logger.info('Using typing_extensions.')
+            return True
+        except ImportError:
+            logger.info('No typing_extensions.')
+            logger.info(PY_VERSION)
+            return PY_VERSION[0] == 3 and PY_VERSION[1] >= 9
```

Comment:
This check is on wrong. It only checks does the running side Python have this functionality, not does the actually tested Robot have it.